tests/operators/test_autodiff.py

```diff
@@ -23,54 +22,65 @@
 
 
 @fixture(scope="module")
-@parametrize(backend=["cufinufft", "finufft"])
-@parametrize(autograd=["data"])
+@parametrize(backend=["cufinufft", "finufft", "gpunufft"])
 @parametrize_with_cases(
     "kspace_loc, shape",
     cases=[
         CasesTrajectories.case_grid2D,
         CasesTrajectories.case_nyquist_radial2D,
-    ],  # 2D cases only for reduced memory footprint.
+        CasesTrajectories.case_nyquist_radial3D_lowmem,
+    ],
 )
-def operator(kspace_loc, shape, backend, autograd):
+def operator(kspace_loc, shape, backend):
     """Create NUFFT operator with autodiff capabilities."""
     kspace_loc = kspace_loc.astype(np.float32)
-
-    nufft = get_operator(backend_name=backend, autograd=autograd)(
+    nufft = get_operator(backend_name=backend, wrt_data=True, wrt_traj=True)(
         samples=kspace_loc,
         shape=shape,
         smaps=None,
         squeeze_dims=False,  # Squeezing breaks dimensions !
     )
-
     return nufft
 
 
-@fixture(scope="module")
```

Comment:
Is there a reason to move away from the fixture setup?

Reply:
Wait this is new to me.. I have no idea... @alineyyy if you are unaware, please add it back..

Reply:
Ohhhhh sorry!! I didn't notice that and I will add it back

Reply:
For this fixture setup, I find that I can't add it back : ( 
I think operator.samples(trajectory) need to be put on device before we get the NDFT matrix. If we want to transform the data to device in function ndft_matrix, we need to add a parameter "interface" which will lead to a failed execution of the tests. 
Do you have any ideas to change here in order to add the setup back? @paquiteau @chaithyagr

Reply:
Thats good catch. I am okay with current state (as long as we run the test), Ill let @paquiteau if he has a better fix